### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#181] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente proprietà sull'anello $(\mathbb{Z},+,\cdot)$:
- $MCD(a,b)=d \Rightarrow MCD(\frac{a}{d}, \frac{b}{d})=1$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/520490d0-545d-4373-9cdc-3afd0993a92d)
